src/schema/markdown/serializer.js

```diff
@@ -48,11 +48,16 @@ export const paragraph = (state, node) => {
   state.closeBlock(node);
 };
 export const image = (state, node) => {
+  let src = state.esc(node.attrs.src);
+  const hasQueryAlready = src && src.indexOf('?') > -1;
+  if (node.attrs.height && !hasQueryAlready) {
+    src += `?cw_image_height=${node.attrs.height}px`;
+  }
```

Comment:
Is it possible to use the URL methods here?

Reply:
@scmmishra Done https://github.com/chatwoot/prosemirror-schema/pull/21/files#diff-df55ca64497de9df74c663f424fc6b01cfcaeb0acb875db40ce8e5a60b047510R53